app/client/components/identity/IdentityLocations.ts

```diff
@@ -0,0 +1,26 @@
+import { conf } from "../../../server/config";
+
+const url = (subdomain: string, domain: string, path?: string) =>
+  `https://${subdomain}.${domain}${path ? path : ""}`;
+
+const DOMAIN =
+  typeof window !== "undefined" && window.guardian
+    ? window.guardian.domain
+    : conf.DOMAIN;
+
+const getIDAPIUrl = () => {
```

Comment:
MINOR: personally would make this a ternary, so the function is just an expression rather than block with a return (and also consistent with the `url` function above)